src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -404,14 +407,20 @@ def get(self, request: Request, project: Project, credentials_id: str) -> Respon
         itunes_connect.load_session_cookie(session, symbol_source_cfg.itunesSession)
         itunes_session_info = itunes_connect.get_session_info(session)
 
-        pending_downloads = AppConnectBuild.objects.filter(project=project, fetched=False).count()
+        builds = AppConnectBuild.objects.filter(project=project)
+        pending_downloads = builds.filter(fetched=False).count()
+        sorted_builds = builds.sort(key=lambda b: b["uploaded_to_appstore"])
+        latest_build_version = (
+            sorted_builds[-1]["bundle_short_version"] if sorted_builds.len > 0 else None
+        )
```

Comment:
tbh, I have no idea what kind of query this generates internally, but would be good to do the sorting in SQL, and then just use `.first()` to fetch only a single row. Also, maybe we want to show both the bundle_short_version as well as the build. Formatted in the way that normal ios builds are formatted.

Reply:
is this formatting done by the UI, or is this something we should be doing on the BE before sending it up to the FE? i.e. do i also send up the build in the payload, or should i do a bit of preprocessing on these values here?

Reply:
yes, let's show both.  I don't think we need to do any formatting of version and build, they're both strings and can be displayed directly.